### ISSUE TITLE
Is this thing on?

### ISSUE THREAD

**Issue by StingyJack** [oldauntie/ababu.win#2]
J/k, kind of...

I saw on your website that big things are coming in 2022, but I don't see any recent commits here. Are you still maintaining this? (I know it can be hard to keep these projects going without help)

**Comment by chrislarabell** [comment]
@StingyJack - looking for vet software, too?

**Comment by StingyJack** [comment]
Yeah, the upgrade coordinator at Patterson/intravet is trying to tell me that a small 3 station practice needs a dedicated Xeon based server and three i5 based workstations just to run a .net+SQL program. She is refusing to run the upgrade because the new hardware does not carry those labels on the CPUs. So I'm looking for options.

**Comment by chrislarabell** [comment]
Yeah, not a huge fan of the available practice management software.  I was trying to get access to an API for one and they just ignored me.  Could probably fork this and work on updates if no response.

**Comment by oldauntie** [comment]
hi there. thank you for your interest. ababu.win is actually in stand by since a more user-friendly and zero-conf and multi-platform clone is on the way (see beta.ababu.cloud for a working demo). As stated on OldAuntie.org website we are looking for any kind of help. Developer and Vets (first of all!) that can be a guidance inspiring and defining project requirements. If you are interested please write back at oldauntie@oldauntie.org
